setup.py

```diff
@@ -46,7 +46,7 @@ def read(*rnames):
     keywords="lock",
     url='https://github.com/zopefoundation/zc.lockfile',
     python_requires='>=3.9',
-    install_requires='setuptools',
+    setup_requires='setuptools',
```

Comment:
Same as in `zope.proxy`: Please do not add `setuptools` to `setup_requires`.

Reply:
thanks for the review